eng/pipelines/runtimelab.yml

```diff
@@ -76,7 +61,7 @@ jobs:
     buildConfig: Release
     platforms:
     - Linux_x64
-    - Windows_NT_x64
+    - FreeBSD
     jobParameters:
       testGroup: innerloop
       buildArgs: -s clr+libs+libs.tests+installer -c $(_BuildConfig) /p:ArchiveTests=true
```

Comment:
Since we're not running tests at the moment you need to remove the `extraStepsTemplate` and `extraStepsParameters` section.